Formula/g/ghc.rb

```diff
@@ -38,73 +29,54 @@ class Ghc < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "python@3.11" => :build
-  depends_on "sphinx-doc" => :build
-  depends_on macos: :catalina
+  depends_on "xz" => :build
```

Comment:
Is this not an implicit dependency from the URL being `.xz`?

Reply:
In the previous ghc build attempt they hit the  `Linux: /bin/sh: 1: xz: not found ` issue, so I copied the dependency from that PR. Let met try without.

Reply:
Ah ok, the build itself maybe uses it somewhere. Make sense, thanks for clarifying.

It used to be the case that implicit dependencies were also in the build environment but we changed that recently.

Reply:
It's possible it might only be needed on Linux, so worth seeing what the CI says.